src/clients/FlexbaseClient.Company.ts

```diff
@@ -20,6 +20,11 @@ interface Payment {
     id: string;
 }
 
+interface PaymentsResponse {
```

Comment:
You could extend this interface with our `FlexbaseResponse` interface, and we can get the `success` and `error` values

```
interface PaymentsResponse extends FlexbaseResponse {
    payments?: Payment[];
};
```
The `payments` is optional if the function catch an error